client/tests/webdriver/baseSpecs/assessmentsForPeople.spec.js

```diff
@@ -28,13 +35,22 @@ describe("For the periodic person assessments", () => {
       Search.linkOfPersonFound(PERSON_SEARCH_STRING).click()
     })
 
-    it("Should allow advisor to successfully add an assessment", () => {
+    it("Should display nested question sets", () => {
       ShowPerson.assessmentsTable.waitForExist()
       ShowPerson.assessmentsTable.waitForDisplayed()
 
       ShowPerson.addPeriodicAssessmentButton.click()
       ShowPerson.waitForAssessmentModalForm()
+      ShowPerson.topLevelQuestionSetTitle.waitForDisplayed()
+      ShowPerson.bottomLevelQuestionSetTitle.waitForDisplayed()
+    })
 
+    it("Should display validation error messages on every level", () => {
+      ShowPerson.saveAssessmentButton.click()
+      expect(ShowPerson.getValidationErrorMessages()).to.eql(ERROR_MESSAGES)
```

Comment:
`to.equal()` can be used to be consistent.

Reply:
They are different actually. `to.equal()` cannot be used here as it checks if they are strictly equal while `to.eql() `checks deep equality.